# University of Texas HDL-64E calibration parameters
lasers:
- {dist_correction: 0.100000001490116, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 0, max_intensity: 255, min_intensity: 0,
  rot_correction: -0.0698131695389748, two_pt_correction_available: false, vert_correction: -0.124932751059532,
  vert_offset_correction: 0}
- {dist_correction: 0.280000001192093, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 1, max_intensity: 255, min_intensity: 0,
  rot_correction: -0.0392699092626572, two_pt_correction_available: false, vert_correction: -0.118993431329727,
  vert_offset_correction: 0}
- {dist_correction: 0.319999992847443, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 2, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.0698131695389748, two_pt_correction_available: false, vert_correction: 0.0055470340885222,
  vert_offset_correction: 0}
- {dist_correction: 0.230000004172325, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 3, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.104719758033752, two_pt_correction_available: false, vert_correction: 0.0114863449707627,
  vert_offset_correction: 0}
- {dist_correction: 0.0700000002980232, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 4, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.013962633907795, two_pt_correction_available: false, vert_correction: -0.113056324422359,
  vert_offset_correction: 0}
- {dist_correction: 0.0900000035762787, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 5, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.0392699092626572, two_pt_correction_available: false, vert_correction: -0.107121199369431,
  vert_offset_correction: 0}
- {dist_correction: 0.119999997317791, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 6, max_intensity: 255, min_intensity: 0,
  rot_correction: 0, two_pt_correction_available: false, vert_correction: -0.148716226220131,
  vert_offset_correction: 0}
- {dist_correction: 0.200000002980232, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 7, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.0226892791688442, two_pt_correction_available: false, vert_correction: -0.142765983939171,
  vert_offset_correction: 0}
- {dist_correction: 0.129999995231628, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 8, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.0820304751396179, two_pt_correction_available: false, vert_correction: -0.101187855005264,
  vert_offset_correction: 0}
- {dist_correction: 0.159999996423721, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 9, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.113446399569511, two_pt_correction_available: false, vert_correction: -0.0952560678124428,
  vert_offset_correction: 0}
- {dist_correction: 0.159999996423721, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 10, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.0698131695389748, two_pt_correction_available: false, vert_correction: -0.136818811297417,
  vert_offset_correction: 0}
- {dist_correction: 0.159999996423721, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 11, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.104719758033752, two_pt_correction_available: false, vert_correction: -0.130874469876289,
  vert_offset_correction: 0}
- {dist_correction: 0.129999995231628, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 12, max_intensity: 255, min_intensity: 0,
  rot_correction: -0.068067841231823, two_pt_correction_available: false, vert_correction: -0.05375986546278,
  vert_offset_correction: 0}
- {dist_correction: 0.200000002980232, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 13, max_intensity: 255, min_intensity: 0,
  rot_correction: -0.0349065847694874, two_pt_correction_available: false, vert_correction: -0.0478330813348293,
  vert_offset_correction: 0}
- {dist_correction: 0.170000001788139, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 14, max_intensity: 255, min_intensity: 0,
  rot_correction: -0.0837758108973503, two_pt_correction_available: false, vert_correction: -0.0893256440758705,
  vert_offset_correction: 0}
- {dist_correction: 0.239999994635582, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 15, max_intensity: 255, min_intensity: 0,
  rot_correction: -0.0479965545237064, two_pt_correction_available: false, vert_correction: -0.0833963677287102,
  vert_offset_correction: 0}
- {dist_correction: 0.180000007152557, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 16, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.00872664619237185, two_pt_correction_available: false, vert_correction: -0.0419059917330742,
  vert_offset_correction: 0}
- {dist_correction: 0.0599999986588955, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 17, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.0392699092626572, two_pt_correction_available: false, vert_correction: -0.035978376865387,
  vert_offset_correction: 0}
- {dist_correction: 0.140000000596046, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 18, max_intensity: 255, min_intensity: 0,
  rot_correction: 0, two_pt_correction_available: false, vert_correction: -0.0774680152535439,
  vert_offset_correction: 0}
- {dist_correction: 0.150000005960464, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 19, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.0305432621389627, two_pt_correction_available: false, vert_correction: -0.0715404152870178,
  vert_offset_correction: 0}
- {dist_correction: 0.219999998807907, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 20, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.0855211317539215, two_pt_correction_available: false, vert_correction: -0.0300500374287367,
  vert_offset_correction: 0}
- {dist_correction: 0.140000000596046, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 21, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.109083078801632, two_pt_correction_available: false, vert_correction: -0.024120757356286,
  vert_offset_correction: 0}
- {dist_correction: 0.0799999982118607, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 22, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.0698131695389748, two_pt_correction_available: false, vert_correction: -0.0656133219599724,
  vert_offset_correction: 0}
- {dist_correction: 0.159999996423721, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 23, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.104719758033752, two_pt_correction_available: false, vert_correction: -0.0596865378320217,
  vert_offset_correction: 0}
- {dist_correction: 0.119999997317791, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 24, max_intensity: 255, min_intensity: 0,
  rot_correction: -0.0610865242779255, two_pt_correction_available: false, vert_correction: 0.0174280721694231,
  vert_offset_correction: 0}
- {dist_correction: 0.219999998807907, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 25, max_intensity: 255, min_intensity: 0,
  rot_correction: -0.0349065847694874, two_pt_correction_available: false, vert_correction: 0.0233724191784859,
  vert_offset_correction: 0}
- {dist_correction: 0.159999996423721, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 26, max_intensity: 255, min_intensity: 0,
  rot_correction: -0.0785398185253143, two_pt_correction_available: false, vert_correction: -0.0181903336197138,
  vert_offset_correction: 0}
- {dist_correction: 0.259999990463257, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 27, max_intensity: 255, min_intensity: 0,
  rot_correction: -0.0471238903701305, two_pt_correction_available: false, vert_correction: -0.0122585473582149,
  vert_offset_correction: 0}
- {dist_correction: 0.140000000596046, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 28, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.0104719763621688, two_pt_correction_available: false, vert_correction: 0.0293195936828852,
  vert_offset_correction: 0}
- {dist_correction: 0.219999998807907, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 29, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.0349065847694874, two_pt_correction_available: false, vert_correction: 0.0352698266506195,
  vert_offset_correction: 0}
- {dist_correction: 0.209999993443489, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 30, max_intensity: 255, min_intensity: 0,
  rot_correction: -0.00436332309618592, two_pt_correction_available: false, vert_correction: -0.00632520206272602,
  vert_offset_correction: 0}
- {dist_correction: 0.140000000596046, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 31, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.0296705979853868, two_pt_correction_available: false, vert_correction: -0.000390077300835401,
  vert_offset_correction: 0}
- {dist_correction: 0.119999997317791, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 32, max_intensity: 255, min_intensity: 0,
  rot_correction: -0.122173048555851, two_pt_correction_available: false, vert_correction: -0.396850973367691,
  vert_offset_correction: 0}
- {dist_correction: 0.0199999995529652, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 33, max_intensity: 255, min_intensity: 0,
  rot_correction: -0.0610865242779255, two_pt_correction_available: false, vert_correction: -0.387918144464493,
  vert_offset_correction: 0}
- {dist_correction: 0.100000001490116, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 34, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.0959931090474129, two_pt_correction_available: false, vert_correction: -0.200955957174301,
  vert_offset_correction: 0}
- {dist_correction: 0.230000004172325, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 35, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.148352980613708, two_pt_correction_available: false, vert_correction: -0.192023113369942,
  vert_offset_correction: 0}
- {dist_correction: 0.170000001788139, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 36, max_intensity: 255, min_intensity: 0,
  rot_correction: -0.00872664619237185, two_pt_correction_available: false, vert_correction: -0.378992766141891,
  vert_offset_correction: 0}
- {dist_correction: 0.150000005960464, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 37, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.0610865242779255, two_pt_correction_available: false, vert_correction: -0.370074152946472,
  vert_offset_correction: 0}
- {dist_correction: 0.0500000007450581, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 38, max_intensity: 255, min_intensity: 0,
  rot_correction: -0.0174532923847437, two_pt_correction_available: false, vert_correction: -0.43128889799118,
  vert_offset_correction: 0}
- {dist_correction: 0.270000010728836, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 39, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.0349065847694874, two_pt_correction_available: false, vert_correction: -0.423701733350754,
  vert_offset_correction: 0}
- {dist_correction: 0.180000007152557, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 40, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.122173048555851, two_pt_correction_available: false, vert_correction: -0.361161530017853,
  vert_offset_correction: 0}
- {dist_correction: 0.189999997615814, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 41, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.174532920122147, two_pt_correction_available: false, vert_correction: -0.352254241704941,
  vert_offset_correction: 0}
- {dist_correction: 0.100000001490116, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 42, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.104719758033752, two_pt_correction_available: false, vert_correction: -0.414742022752762,
  vert_offset_correction: 0}
- {dist_correction: 0.200000002980232, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 43, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.165806278586388, two_pt_correction_available: false, vert_correction: -0.405792057514191,
  vert_offset_correction: 0}
- {dist_correction: 0.219999998807907, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 44, max_intensity: 255, min_intensity: 0,
  rot_correction: -0.113446399569511, two_pt_correction_available: false, vert_correction: -0.28999200463295,
  vert_offset_correction: 0}
- {dist_correction: 0.200000002980232, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 45, max_intensity: 255, min_intensity: 0,
  rot_correction: -0.0567231997847557, two_pt_correction_available: false, vert_correction: -0.281101644039154,
  vert_offset_correction: 0}
- {dist_correction: 0.150000005960464, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 46, max_intensity: 255, min_intensity: 0,
  rot_correction: -0.13962633907795, two_pt_correction_available: false, vert_correction: -0.343351542949677,
  vert_offset_correction: 0}
- {dist_correction: 0.219999998807907, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 47, max_intensity: 255, min_intensity: 0,
  rot_correction: -0.0872664600610733, two_pt_correction_available: false, vert_correction: -0.334452718496323,
  vert_offset_correction: 0}
- {dist_correction: 0.0599999986588955, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 48, max_intensity: 255, min_intensity: 0,
  rot_correction: 0, two_pt_correction_available: false, vert_correction: -0.272210210561752,
  vert_offset_correction: 0}
- {dist_correction: 0.129999995231628, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 49, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.0558505356311798, two_pt_correction_available: false, vert_correction: -0.26331701874733,
  vert_offset_correction: 0}
- {dist_correction: 0.00999999977648258, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 50, max_intensity: 255, min_intensity: 0,
  rot_correction: -0.0174532923847437, two_pt_correction_available: false, vert_correction: -0.323895305395126,
  vert_offset_correction: 0}
- {dist_correction: 0.150000005960464, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 51, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.0349065847694874, two_pt_correction_available: false, vert_correction: -0.316663861274719,
  vert_offset_correction: 0}
- {dist_correction: 0.200000002980232, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 52, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.122173048555851, two_pt_correction_available: false, vert_correction: -0.254421383142471,
  vert_offset_correction: 0}
- {dist_correction: 0.239999994635582, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 53, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.165806278586388, two_pt_correction_available: false, vert_correction: -0.245522528886795,
  vert_offset_correction: 0}
- {dist_correction: 0.180000007152557, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 54, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.104719758033752, two_pt_correction_available: false, vert_correction: -0.307772427797318,
  vert_offset_correction: 0}
- {dist_correction: 0.219999998807907, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 55, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.157079637050629, two_pt_correction_available: false, vert_correction: -0.298882067203522,
  vert_offset_correction: 0}
- {dist_correction: 0.140000000596046, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 56, max_intensity: 255, min_intensity: 0,
  rot_correction: -0.104719758033752, two_pt_correction_available: false, vert_correction: -0.183082059025764,
  vert_offset_correction: 0}
- {dist_correction: 0.319999992847443, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 57, max_intensity: 255, min_intensity: 0,
  rot_correction: -0.0610865242779255, two_pt_correction_available: false, vert_correction: -0.17413204908371,
  vert_offset_correction: 0}
- {dist_correction: 0.200000002980232, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 58, max_intensity: 255, min_intensity: 0,
  rot_correction: -0.13962633907795, two_pt_correction_available: false, vert_correction: -0.236619830131531,
  vert_offset_correction: 0}
- {dist_correction: 0.25, dist_correction_x: 0, dist_correction_y: 0, focal_distance: 0,
  focal_slope: 0, laser_id: 59, max_intensity: 255, min_intensity: 0, rot_correction: -0.0785398185253143,
  two_pt_correction_available: false, vert_correction: -0.22771255671978, vert_offset_correction: 0}
- {dist_correction: 0.170000001788139, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 60, max_intensity: 255, min_intensity: 0,
  rot_correction: 0, two_pt_correction_available: false, vert_correction: -0.16517236828804,
  vert_offset_correction: 0}
- {dist_correction: 0.230000004172325, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 61, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.0436332300305367, two_pt_correction_available: false, vert_correction: -0.156202226877213,
  vert_offset_correction: 0}
- {dist_correction: 0.150000005960464, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 62, max_intensity: 255, min_intensity: 0,
  rot_correction: -0.0174532923847437, two_pt_correction_available: false, vert_correction: -0.218799933791161,
  vert_offset_correction: 0}
- {dist_correction: 0.230000004172325, dist_correction_x: 0, dist_correction_y: 0,
  focal_distance: 0, focal_slope: 0, laser_id: 63, max_intensity: 255, min_intensity: 0,
  rot_correction: 0.0314159244298935, two_pt_correction_available: false, vert_correction: -0.209881335496902,
  vert_offset_correction: 0}
num_lasers: 64
distance_resolution: 0.002
